### DISCUSSION TITLE
Next release of sops

### DISCUSSION THREAD

**Discussion by sethiyash** [getsops/sops#1239] [General]
When we can expect the next release of Sops? I'm looking for a patch release with the necessary dependencies bump especially Golang

**Comment by hiddeco** [comment]
>  Given this, it will take a bit of time to adjust the repositories (including the release automation), before getting started with actually rolling out the long overdue patch release.
> [...]
> Please bear with us while we try to work through everything (including issues), this may take some time.
>
> &mdash; https://github.com/getsops/sops/discussions/927#discussioncomment-6314749

As this was posted about a week ago, and we are still working on getting the release automation working again. Please be patient and give us some time to properly get the project up to speed again. We want to ensure that the upcoming release is stable and reliable, while it has practically been resurrected from near dead.

I can't promise any exact date at present, but as you can see from the activities in the repository we have been working on pushing it forward every available moment of time we have had for the last week.

Thanks for your comprehension. :cake:

**Comment by sethiyash** [comment]
@hiddeco Thanks for the update.
